src/coreclr/src/tools/ReadyToRun.SuperIlc/BuildFolderSet.cs

```diff
@@ -252,7 +252,7 @@ public bool CompileFramework()
                 return true;
             }
 
-            Stopwatch stopwatch = new Stopwatch();
+            var stopwatch = new Stopwatch();
             stopwatch.Start();
```

Comment:
Nit: this could be:
```C#
Stopwatch stopwatch = Stopwatch.StartNew();
```
Same for another occurrence later.

Reply:
@stephentoub, Thanks! I've applied your suggestion.